src/views/dataset/NewDataset.vue

```diff
@@ -372,6 +386,22 @@ export default class NewDataset extends Vue {
   }
 
   filesChanged(files: FileUpload[]) {
+    const totalSize = files.reduce((sum, { file }) => sum + file.size, 0);
+    const maxSizeBytes = this.maxTotalFileSize * 1024 * 1024; // Convert MB to bytes
+
+    if (totalSize > maxSizeBytes) {
+      this.fileSizeExceeded = true;
+      this.fileSizeExceededMessage = `Total file size (${(totalSize / 1024 / 1024).toFixed(1)} MB) exceeds the maximum allowed size of ${this.maxTotalFileSize} MB`;
```

Comment:
I would expect to see that error string inside the `v-alert` component.  
Here it's a bit hidden.

You would need to set `totalSize` as a reactive value of the component to do that

Reply:
Good point, thanks!